runs/models.tsp

```diff
@@ -84,7 +139,32 @@ model CreateThreadAndRunRequest {
    * behavior on a per-run basis.
    */
   // !no_nullable! @maxItems(20)
```

Comment:
```suggestion
  // !no_nullable! 
  @maxItems(20)
```
Spec says it is nullable. What do we do with the !no_nullable! comment?
![image](https://github.com/joseharriaga/openai-in-typespec/assets/21109913/cafe1755-e9a3-4da0-a4fd-23cd0d958661)

Reply:
These "no_nullable" workarounds for decorators should all be gone now with the extracted types for nullable complex arrays.